vuepress/docs/next/docs/reference/backup-restore.md

```diff
@@ -0,0 +1,88 @@
+
+```
+kubectl cp quickstart-deployment:/tmp/entando-data.tar.gz entando-data.tar.gz
+```
+
+3. Push it to the restored instance:
+
+```
+kubectl cp entando-data.tar.gz quickstart-deployment:/tmp/entando-data.tar.gz
+```
+
+4. Extract the files to the restored environment:
+```
+cd /entando-data
+tar xfvz /tmp/entando-data.tar.gz --strip-components=1
+```
+
+## Backup and Restore Keycloak
+Entando can auto-create a new Keycloak instance but previous secrets, clients, and permissions will be replaced and recreated from scratch. A less disruptive path may be to backup and restore EntandoApp’s realm to the new Keycloak instance so there is no data loss. For detailed instructions, follow the [Backing Up and Restoring Keycloak](../../tutorials/devops/backing-restoring-keycloak.md) tutorial. If only the EntandoApp is restored and the original Keycloak is used, additional Keycloak work is not required.
+
+## General Guidelines for Bundles
+Bundles should be reinstalled when restoring from a backup with the strategy provided by the bundle creator. Below are some suggested guidelines. 
+
+1. Export all the EntandoDeBundle custom resources from the source instance and restore them to the new instance.
+Then manually install each bundle. The reinstallation will create the proper client credentials and permissions required by the bundle.
+2. If custom roles and permissions were applied in the original instance, manually transfer these as well. Note, if a different Keycloak is used for the restored instance, then the client and permissions for the bundle may need to be exported/imported to the new instance. See the [Backing Up and Restoring Keycloak](../../tutorials/devops/backing-restoring-keycloak.md) tutorial for details.
+3. For Bundle DB schemas, different methods are required for auto-provisioned and external databases.
+
+    A. For auto-provisiond DBMS schemas:  
+             1. When the provisioning is complete, scale down the plugin deployment to 0. Restore the source data with the database-native backup/restore tools and then scale up the deployment.  
+             2. In case of GCG-generated Spring Boot plugins, the plugin's application will autonomously apply the Liquibase data migration scripts.   
+             3. In order to connect to an internal auto-provisioned DBMS, use kubectl port-forward against the bundle DB.   
+
+    B. For External DBMS schemas:  
+       1. Export all the EntandoDeBundle CRs from the source instance and restore them to the target instance.  
+       2. Then create the restored instance DB schema by making a copy of the source instance DB schema with the DB native backup/restore tools.  
+       3. Copy DB credential secrets from the original instance into the restored instance.  
+       4. Reinstall the bundles.   
+
+
+### Private Repository Bundles
+If a bundle originated from a private repository in the original instance, the secrets, configurations, etc. will need to be replicated in the new instance. 
```

Comment:
originated/original very close together; "source instance" like referred to above? (for consistency and to avoid repetition)

Secrets

etc., (comma is missing)

Reply:
just keeping this one because it relates to another lowercase secrets